apps/researcher/src/app/[locale]/objects/[id]/page.tsx

```diff
@@ -141,7 +141,7 @@ export default async function Details({params}: Props) {
                 key={enrichment.id}
                 className="border-r border-consortiumBlue-400 mr-4 pr-4"
               >
-                <div className="">{enrichment.description}</div>
+                <div>{enrichment.description}</div>
                 <div className="text-xs font-normal  hidden sm:block text-consortiumBlue-100">
```

Comment:
Very, very minor:
1. There's a space too many
2. What's our convention for class names? Tailwind uses lowercase-with-dashes, but `consortiumBlue` deviates from this notation.

Reply:
1. I have removed this white space
2. Good point. All our custom colors are camel case. I don't want to change this in this pull request. But I will discuss this with Bas.